evergreen/eg_app/consumers.py

```diff
@@ -0,0 +1,99 @@
+import json
+
+# from asgiref.sync import async_to_sync
+from channels.db import database_sync_to_async
+from channels.generic.websocket import AsyncWebsocketConsumer
+
+# from channels.layers import get_channel_layer
+from django.apps import apps
+
+
+class FeedConsumer(AsyncWebsocketConsumer):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.room_group_name = "feed"
+
+    # connects to socket
+    async def connect(self):
+
+        print("WebSocket connection attempt received!")
+
+        # Add to group
+        await self.channel_layer.group_add(self.room_group_name, self.channel_name)
+        await self.accept()
+        print("(GOOD) WebSocket connection accepted successfully!")
+
+    # disconnects connection handler
+    async def disconnect(self, close_code):
+
+        await self.channel_layer.group_discard(self.room_group_name, self.channel_name)
+        print(f"(BAD) WebSocket disconnected with code: {close_code}")
+
+    # updated to all clients
+    async def feed_update(self, event):
+
+        # sending a lovey message to websocket
+        await self.send(text_data=json.dumps(event["data"]))
+        print("Feed update sent successfully!")
+
+    # handle recieving like events!
```

Comment:
I before E except after C (this is not one of the exceptions)
```suggestion
    # handle receiving like events!
```